test/unittests/mongoBackend/mongoRegisterContext_update_test.cpp

```diff
@@ -908,11 +910,6 @@ TEST(mongoRegisterContext_update, NotifyContextAvailability1)
           .Times(0);
   setNotifier(notifierMock);
 
-  TimerMock* timerMock = new TimerMock();
```

Comment:
This seems to be a change in the unit test logic... Or maybe is a mock not needed in this test?